### ISSUE TITLE
Task view moves after task adding

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bardsoftware/ganttproject#550]
```
What steps will reproduce the problem?
1. Open file Gantt_moving.gan. 
2. Select task "Zimní semestr". Have some zoom in Gantt diagram.
3. Remember which is the highest task you can see now (in screenshot 
Gantt_moving1.png it is "Rozvrhy - sestavení, přednášející".
4. Press CTRL-T (new task). 
5. Screenshot Gantt_moving2.png was made right after step 4. Now is the highest 
task the new task.

What is the expected output? What do you see instead?
Expected output is new task under the selected task and I should can see the 
same highest task as it was in step 3. But task window moves which is annoying.

What version of the product are you using? On what operating system?
GanttProject 2.5.3 Praha (build 1142), Win 7 64b Eng.

Please provide any additional information below.
I cannot reproduce this behavior in another GanttProject file so there is 
Gantt_moving.gan in attachment.

```

Original issue reported on code.google.com by `i.hate.this.mail.asdf` on 13 Jul 2012 at 3:31

Attachments:
- [Gantt_moving.gan](https://storage.googleapis.com/google-code-attachments/ganttproject/issue-550/comment-0/Gantt_moving.gan)
- [Gantt_moving1.png](https://storage.googleapis.com/google-code-attachments/ganttproject/issue-550/comment-0/Gantt_moving1.png)
- [Gantt_moving2.png](https://storage.googleapis.com/google-code-attachments/ganttproject/issue-550/comment-0/Gantt_moving2.png)
